renovate.json

```diff
@@ -5,6 +5,17 @@
   ],
   "dependencyDashboard": false,
   "prConcurrentLimit": 40,
+  "packageRules": [
+    {
+      "groupName": "github-actions",
+      "matchManagers": [
+        "github-actions"
+      ],
+      "schedule": [
+        "on sunday"
+      ]
```

Comment:
Open for discussion: do we want updates less often, e.g. just monthly? Do we want to keep some form of automerge (maybe just minor and patch version updates)?